src/Detail.js

```diff
@@ -1,9 +1,17 @@
 import React from 'react';
+import Description from './Description';
 import './Detail.css';
+import Poster from './Poster';
+import Button from './Button';
 
 function Detail() {
   return (
-   
+   /* event listener for card, display card */ 
+   <div>
+       <Poster/>
```

Comment:
Interesting choice to make these 3 separate components, rather than one MovieCard type component. It's not wrong - just different from how I've seen it in the past! I'm curious what your though process was here.